File: ALearn/cacbaitapc/basic/1000 bai code/bai42_61/bai42.cpp
//Cho n là số nguyên dương. Hãy tìm giá trị nguyên dương k lớn nhất sao
//cho S(k) < n. Trong đó chuỗi S(k) được định nghĩa như sau : S(k) = 1 +
//2 + 3 + … + k
#include <conio.h>
#include <stdio.h>

int main(){
    unsigned int n;
    printf("enter a positive integer:");
    scanf("%u",&n);
    int s=1;
    for(int i=2;i<=n/2;i++){
        s+=i;
       if(s>=n){
           s-=i;
           break;
       } 
    }
    printf("s(k)=%d",s);
    getch();
}

File: learn/python/100baicode/41-60/bai42.py
# Viết một chương trình để tạo tuple khác, chứa các giá trị là số chẵn trong 
# tuple (1,2,3,4,5,6,7,8,9,10) cho trước.
t1 = (1,2,3,4,5,6,7,8,9,10)
t2 = (i for i in t1 if i % 2 == 0 )
print (tuple(t2))